docs/migrate-from-masb/mssql.md

```diff
@@ -0,0 +1,67 @@
+# Migrate [Azure SQL Database](https://azure.microsoft.com/en-us/services/sql-database/) Service Instances From MASB To OSBA
+
+***Note***: this guidance is specially for existing server scenario. Straighter, the SQL Databases created by MASB must be on a server whose credentials are provided from MASB manifest.
```

Comment:
Why the guidance doesn't apply to new server scenario?

Reply:
Most MASB users use existing servers. For new server scenario, MASB doesn't delete the server when deprovisioning, either. So, this guidance should almost work for them but some more steps to get server credentials. The extra steps are not very brief. We can provide a single answer then if someone asks for it.